Jenkinsfile

```diff
@@ -1,5 +1,113 @@
+// Copyright 2018 SUSE LINUX GmbH, Nuernberg, Germany.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
 def targetBranch = env.getEnvironment().get('CHANGE_TARGET', env.BRANCH_NAME)
 
 library "kubic-jenkins-library@${targetBranch}"
 
-coreKubicProjectCi()
+// TODO: Move this whole check into a function
+if (env.CHANGE_AUTHOR != null) {
+    // TODO: Don't hardcode salt repo name, find the right place
```

Comment:
https://wiki.jenkins.io/display/JENKINS/Active+Choices+Plugin 
maybe?

Reply:
This TODO comes from a copy and paste from the `jenkins-library` repo - the real TODO is to not copy and paste the code :)